tests/common_testing/alfasim_sdk_common_testing/case_builders.py

```diff
@@ -0,0 +1,106 @@
+from barril.units import Array
+
+from alfasim_sdk.alfacase import case_description
+
+
+def build_simple_segment():
+    """
+    Return an instance of PipeSegmentsDescription with a pre-populated value.
+
+    The pre-filled value was the default value used by all tests before refactoring,
+    which removed `diameter` and `absolute_roughness` from PipeDescription.
+    """
+
```

Comment:
Very deeply minor: If you change anything else, consider removing those empty lines (Or are those added by the fix format?)

Reply:
Sure, https://github.com/ESSS/alfasim-sdk/pull/108/commits/4e5a42da32dc526f8bfad3877ce01a266b39d7b7 fix this ;)